js/private/js_run_devserver.mjs

```diff
@@ -34,13 +37,9 @@ function mkdirpSync(p) {
 }
 
 // Determines if a file path refers to a node module.
-//
```

Comment:
Any reason to change this? Other then adding the `export` keyword...

Reply:
export added so it can be unit tested and changed the parameter name just for consistency